src/Optimization/hiopAlgFilterIPM.cpp

```diff
@@ -1457,12 +1520,12 @@ hiopSolveStatus hiopAlgFilterIPMQuasiNewton::run()
 
 void hiopAlgFilterIPMQuasiNewton::outputIteration(int lsStatus, int lsNum, int use_soc, int use_fr)
 {
-  if(iter_num/10*10==iter_num)
+  if(iter_num_/10*10==iter_num_)
```

Comment:
should we use iter_num_total_ here?

Reply:
no, it will re-print header every 10 iterations, when restarting this may not happen at multiples of 10.